modules/ROOT/pages/publish-apis.adoc

```diff
@@ -15,7 +15,7 @@ will lead the CMS article release process.
 
 == Add APIs from Exchange to Your Community
 
-. Open the ACM Control Panel and select *Manage APIs*. From this tab, you can manage the APIs in your community, including their metadata and visibility.
+. Open the ACM control panel and select *Manage APIs*. From this tab, you can manage the APIs in your community, including their metadata and visibility.
```

Comment:
GLOBAL:
Steps should be one sentence maximum. Place additional info, if NECESSARY, below:

. Open the ACM control panel and select *Manage APIs*. 
  From this tab, you can manage the APIs in your community, including their metadata and visibility.